app/private-cloud/products/(product)/[licencePlate]/comments/page.tsx

```diff
@@ -65,17 +66,14 @@ export default privateCloudProductComments(({ pathParams, queryParams, session }
       {comments?.length > 0 ? (
         <ul>
           {comments.map((comment: Comment) => (
-            <li key={comment.id}>
-              <p>
-                <strong>Comment:</strong> {comment.text}
-              </p>
-              <p>
-                <strong>By:</strong> {comment.user.firstName} {comment.user.lastName} ({comment.user.email})
-              </p>
-              <p>
-                <strong>Posted on:</strong> {new Date(comment.created).toLocaleString()}
-              </p>
-            </li>
+            <CommentBubble
+              key={comment.id}
+              text={comment.text}
+              timestamp={new Date(comment.created)}
+              firstName={comment.user.firstName}
+              lastName={comment.user.lastName}
+              isUser={userId === comment.userId}
```

Comment:
`isUser` seems not intuitive name I guess, `isAuthor` or `isWriter` could be better options.